.github/workflows/ci.yml

```diff
@@ -117,15 +129,3 @@ jobs:
 
     - name: Docs
       run: cargo doc --features docs
-
```

Comment:
why remove clippy?

Reply:
I don't think there is any point in leaving this comment out. If you need a clippy later, you can just add it again.

Reply:
right, makes sense